src/test/java/me/botsko/prism/Il8nTest.java

```diff
@@ -0,0 +1,21 @@
+package me.botsko.prism;
+
+import junit.framework.TestCase;
+
+/**
+ * Created for use for the Add5tar MC Minecraft server
+ * Created by benjamincharlton on 2/09/2020.
+ */
+public class Il8nTest extends TestCase {
+
+
+    public void testGetMessage() {
+    }
+
+    public void testGetRawMessage() {
+        assertEquals("Prism - originally by Viveleroi; maintained by <author> v<version>", Il8n.getRawMessage("about-header"));
+    }
+
+    public void testFormatMessage() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6279951)